### PR TITLE
fix: Low contrast issues in kebab menu when using high contrast themes

### PR DESCRIPTION
**Pull request by katydecorah**
#### Details

To fix the low contrast text in the kebab menu when using high contrast mode in web and unified, we could leverage FluentUI styling of the context menu and kebab button to remove added scss to handle high contrast mode.

These changes will have slight UI changes:

- Each menu item's icons are colored blue which is consistent with other FluentUI elements.
- The context menu has a slightly less border-radius as defined by FluentUI.
- FluentUI's context menu items do not have hover state in high contrast mode by default. (🎀Note to self: Open issue in FluentUI). This pull request will set a hover state with `color: highlight` (a pattern in FluentUI) instead of setting `background-color`. 

The table below shows screenshots of the current kebab menu and proposed to illustrate the UI changes described above:

Current | Proposed | Proposed in high contrast
---|---|---
 |  | 

The table below shows screenshots of the fixed high contrast issues in each theme for web and unified. The first menu item is hovered. I also checked that the hovered text has a passing color contrast ratio:

High contrast theme | Web | Unified
|---|---|---
Aquatic |   | 
Desert |  | 
Dusk |   | 
Night sky |  | 

##### Additional changes

- Replace deprecated `name` with `text` in menu items.
- Remove unused color variables.
- By using FluentUI's styling of the context menu, there is no longer gaps in the border edges  in high contrast mode:
Screenshot of context menu where the white border is clipped at the rounded edges

##### Motivation

Addresses #4616

##### Context

- I considered using FluentUI's `MoreVertical` icon and while it would reduce tech debt, the dots in the icon were smaller and felt lower in contrast to the current icon overall. 

#### Pull request checklist

- [x] Addresses an existing issue: #4616
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/common/components/cards/card-kebab-menu-button.scss [hunk 1]
```diff
@@ -1,86 +1,14 @@
...
-        }
+    :global(.ms-Icon) {
+        font-size: 18px; // Increase kebab icon size
     }
 
-    > span {
-        justify-content: center;
+    &:hover,
+    &:global(.is-expanded) {
+        background-color: #edebe9;
```
Comment (katydecorah) [inline src/common/components/cards/card-kebab-menu-button.scss:12]: Note: It's not clear to me why these styles aren't cascading from [FluentUI button](https://developer.microsoft.com/en-us/fluentui#/controls/web/button). Without this line the background color is set to `transparent`.

Comment (katydecorah) [reply src/common/components/cards/card-kebab-menu-button.scss:12]: Note to self: swap for a color variable.

File: src/common/icons/more-actions-menu-icon.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ export const MoreActionsMenuIcon = NamedFC('MoreActionsMenuIcon', () => (
         height="16"
         viewBox="0 0 10 16"
         fill="none"
+        stroke="currentColor"
```
Comment (katydecorah) [inline src/common/icons/more-actions-menu-icon.tsx:13]: By setting `currentColor`, the SVG will inherit the color from the parent -- this works great for high contrast mode.

**Review by JGibson2019 — APPROVED** [review]

**Comment by JGibson2019** [comment]
@katydecorah in HC do we need the border around the kebab menu button itself or is it fine to just have the high contrast color applied to the three dots? I noticed in non-HC mode, both the original and proposed UI feature a grey button that provides some contrast to the white background.

**Comment by katydecorah** [comment]
> @katydecorah  in HC do we need the border around the kebab menu button itself or is it fine to just have the high contrast color applied to the three dots? I noticed in non-HC mode, both the original and proposed UI feature a grey button that provides some contrast to the white background.


@JGibson2019 I checked high contrast mode on the [FluentUI buttons](https://developer.microsoft.com/en-us/fluentui#/controls/web/button) page and the hover/active states are only a color change to the text and icons. I haven't found a source about best practices for high contrast mode with regards to hover, so it's not clear to me if we need an additional hover indicator (in this case highlighting the button border).

**Comment by JGibson2019** [comment]
> > @katydecorah  in HC do we need the border around the kebab menu button itself or is it fine to just have the high contrast color applied to the three dots? I noticed in non-HC mode, both the original and proposed UI feature a grey button that provides some contrast to the white background.
> 
> @JGibson2019 I checked high contrast mode on the [FluentUI buttons](https://developer.microsoft.com/en-us/fluentui#/controls/web/button) page and the hover/active states are only a color change to the text and icons. I haven't found a source about best practices for high contrast mode with regards to hover, so it's not clear to me if we need an additional hover indicator (in this case highlighting the button border).

Gotcha! Do you think we'll need to verify this before closing out #4616?

**Comment by katydecorah** [comment]
>> > @katydecorah  in HC do we need the border around the kebab menu button itself or is it fine to just have the high contrast color applied to the three dots? I noticed in non-HC mode, both the original and proposed UI feature a grey button that provides some contrast to the white background.
> 
>> @JGibson2019 I checked high contrast mode on the [FluentUI buttons](https://developer.microsoft.com/en-us/fluentui#/controls/web/button) page and the hover/active states are only a color change to the text and icons. I haven't found a source about best practices for high contrast mode with regards to hover, so it's not clear to me if we need an additional hover indicator (in this case highlighting the button border).

> Gotcha! Do you think we'll need to verify this before closing out #4616? 

Confirmed with PM that we will not need to add a border on hover since on hover the cursor pointer changes and the color change contrast is 3:1 😎

**Pull request merged**